cmake/CPackAdditionalInstall.cmake

```diff
@@ -0,0 +1,2 @@
+file(INSTALL ${CPACK_ADDITIONAL_INSTALL_RESOURCES} DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
+file(INSTALL ${CPACK_ADDITIONAL_EXAMPLE_APP} DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/example-app")
```

Comment:
Why is this needed?

Reply:
This is the path of the example-app to be shipped in the CPACK package